meta-balena-common/recipes-connectivity/openssh/openssh_%.bbappend

```diff
@@ -0,0 +1,37 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/balena-files:"
+
+SRC_URI_append = " \
+    file://ssh_keys_merger \
+    file://ssh.service \
+"
+
+SYSTEMD_SERVICE_${PN}-sshd += "sshdgenkeys.service"
+
+FILES_${PN}-sshd += " \
+    ${sysconfdir}/avahi/services/ssh.service \
```

Comment:
should this file live in this recipe or avahi?

Reply:
The ssh service is provided by openssh. It's like saying do install a systemd service from the package that starts or from systemd?